lib/msf/base/sessions/meterpreter_options.rb

```diff
@@ -71,6 +71,14 @@ def initialize(info = {})
               'AutoUnhookProcess',
               [true, "Automatically load the unhook extension and unhook the process", false]
             ),
+            OptBool.new(
+              'MeterpreterDebugBuild',
+              [true, 'Use a debug version of Meterpreter', false]
+            ),
+            OptString.new(
```

Comment:
Potentially we want a new option for this, so that it can be validated upfront before the module is ran, example:
https://github.com/rapid7/metasploit-framework/blob/a59aa864c3058aff2adab2db9313d16ee6b8d803/lib/msf/core/opt_enum.rb

Reply:
We have examples of unit testing these opt objects as well